aspnetcore/security/anti-request-forgery.md

```diff
@@ -41,30 +40,67 @@ An example of a CSRF attack:
 Notice that the form action posts to the vulnerable site, not to the malicious site. This is the “cross-site” part of CSRF.
 
 4. The user clicks the submit button. The browser automatically includes the authentication cookie for the requested domain (the vulnerable site in this case) with the request.
+5. The request runs on the server with the user’s authentication context and can do anything that an authenticated user is allowed to do.
 
-5. The request runs on the server with the user’s authentication context, and can do anything that an authenticated user is allowed to do.
+This example requires the user to click the form button. The malicious page could run a script that automatically submits the form or sends a form submission as an AJAX request. The form could  be hidden using CSS so the user never realizes it's present. Using SSL does not prevent a CSRF attack, the malicious site can send an `https://` request. 
```

Comment:
`The form could  be` has an extra space. Few places in this doc has the same. I'd do a find and replace for two spaces that aren't after a period.